kairon/shared/utils.py

```
@@ -159,6 +159,19 @@ def check_empty_string(value: str):
         else:
             return False
 
+    @staticmethod
+    def get_filename_from_url(url: str = None):
```

Ruff (0.14.2)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)